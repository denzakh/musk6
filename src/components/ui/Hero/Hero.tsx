import { FC, ReactNode } from 'react'
import Img from './cover.jpg'

interface HeroProps {
  children?: ReactNode
}

export const Hero: FC<HeroProps> = () => {
  return (
    <div className="hero bg-base-200 min-h-[800px] py-8">
      <div className="hero-content flex-col lg:flex-row-reverse gap-x-8">
        <img src={Img} className="rounded-lg shadow-2xl" />
        <div>
          <h1 className="text-3xl md:text-5xl font-normal font-header">Trilogía clásica!</h1>
          <div className="max-w-[500px] py-6">
            <p className="py-2">
              La Primera trilogía de Star Wars es el primer conjunto de películas pertenecientes a
              la franquicia Star Wars, una ópera espacial estadounidense creada por George Lucas.
            </p>
            <p className="py-2">
              Edición clásica, complementada y restaurada. Star Wars Episodios 4-6 (Blu-ray +
              Blu-ray Extras) (6 discos) (Ediciones remasterizadas): La Guerra de las Galaxias, El
              Imperio Contraataca, El Retorno del Jedi
            </p>
          </div>
          <button className="btn btn-primary btn-xl">
            Comprar <span className="hidden sm:inline-block">con descuento</span>
          </button>
        </div>
      </div>
    </div>
  )
}
