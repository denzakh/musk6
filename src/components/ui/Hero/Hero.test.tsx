import { it, expect } from 'vitest'
import { render } from '@testing-library/react'
import { Hero } from './Hero'

it('Snapshot', () => {
  const comp = render(<Hero />)
  expect(comp).toMatchSnapshot()
})
