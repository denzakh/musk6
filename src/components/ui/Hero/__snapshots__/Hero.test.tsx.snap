// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="hero bg-base-200 min-h-[800px] py-8"
      >
        <div
          class="hero-content flex-col lg:flex-row-reverse gap-x-8"
        >
          <img
            class="rounded-lg shadow-2xl"
            src="/src/components/ui/Hero/cover.jpg"
          />
          <div>
            <h1
              class="text-3xl md:text-5xl font-normal font-header"
            >
              Trilogía clásica!
            </h1>
            <div
              class="max-w-[500px] py-6"
            >
              <p
                class="py-2"
              >
                La Primera trilogía de Star Wars es el primer conjunto de películas pertenecientes a la franquicia Star Wars, una ópera espacial estadounidense creada por George Lucas.
              </p>
              <p
                class="py-2"
              >
                Edición clásica, complementada y restaurada. Star Wars Episodios 4-6 (Blu-ray + Blu-ray Extras) (6 discos) (Ediciones remasterizadas): La Guerra de las Galaxias, El Imperio Contraataca, El Retorno del Jedi
              </p>
            </div>
            <button
              class="btn btn-primary btn-xl"
            >
              Comprar 
              <span
                class="hidden sm:inline-block"
              >
                con descuento
              </span>
            </button>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="hero bg-base-200 min-h-[800px] py-8"
    >
      <div
        class="hero-content flex-col lg:flex-row-reverse gap-x-8"
      >
        <img
          class="rounded-lg shadow-2xl"
          src="/src/components/ui/Hero/cover.jpg"
        />
        <div>
          <h1
            class="text-3xl md:text-5xl font-normal font-header"
          >
            Trilogía clásica!
          </h1>
          <div
            class="max-w-[500px] py-6"
          >
            <p
              class="py-2"
            >
              La Primera trilogía de Star Wars es el primer conjunto de películas pertenecientes a la franquicia Star Wars, una ópera espacial estadounidense creada por George Lucas.
            </p>
            <p
              class="py-2"
            >
              Edición clásica, complementada y restaurada. Star Wars Episodios 4-6 (Blu-ray + Blu-ray Extras) (6 discos) (Ediciones remasterizadas): La Guerra de las Galaxias, El Imperio Contraataca, El Retorno del Jedi
            </p>
          </div>
          <button
            class="btn btn-primary btn-xl"
          >
            Comprar 
            <span
              class="hidden sm:inline-block"
            >
              con descuento
            </span>
          </button>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
